js/tests/unit/tooltip.js

```diff
@@ -680,6 +680,42 @@ $(function () {
     $styles.remove()
   })
 
+  test('should not be adjust position when scrolling', function () {
```

Comment:
@ericmartel So, would I be correct in summarizing this test as "scrolling but still remaining on-screen should not cause viewport-related position adjustments"?